src/screens/DataInputScreen.tsx

```diff
@@ -0,0 +1,65 @@
+
+import React, { useState } from 'react';
+import { View, TextInput, Button } from 'react-native';
+import { DataInputRequest } from '../types/UserTypes';
+import { dataInputApi } from '../apis/DataInputApi';
```

Comment:
There is no `dataInputApi` method. Use `postDataInput` method from '../apis/DataInputApi'